python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -160,7 +160,17 @@ class RandomForestRegressor(Base):
     used in scikit-learn. By default, the cuML Random Forest uses a
     histogram-based algorithms to determine splits, rather than an exact
     count. You can tune the size of the histograms with the n_bins parameter.
-    The instances of RandomForestRegressor cannot be pickled currently.
+
+    **Known Limitations**: This is an initial release of the cuML
+    Random Forest code. It contains a few known limitations:
+
+       * Inference/prediction takes place on the CPU. A GPU-based inference
+         solution based on the forest inference library is planned for a
+         near-future release.
+
+       * Instances of RandomForestRegressor cannot be pickled currently.
+
+
```

Comment:
i believe we need one blank line before the examples, I think there is an extra blank line here.

Reply:
Thanks! Just updated to one blank line. Does it look right now?

Reply:
yes 👍